### PR TITLE
Add artifact schema and tests

### PR DESCRIPTION
**Pull request by sajayantony**
Signed-off-by: Sajay Antony <sajaya@microsoft.com>

### CODE REVIEW

**Comment by sajayantony** [comment]
Staging the PR for schema and tests artifact manifest. We can consider taking this as well. 
/cc @sudo-bmitch @jdolitsky

**Review by sudo-bmitch — COMMENTED** [review]
Should this have the `refers` field, in both the schema and test?

**Comment by sajayantony** [comment]
@sudo-bmitch - just added those. 
I can make a separate PR for image spec modifications.

File: schema/artifact-manifest-schema.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
+  "description": "OpenContainer Artifact Manifest Specification",
+  "$schema": "https://json-schema.org/draft-04/schema#",
```
Comment (sudo-bmitch) [inline schema/artifact-manifest-schema.json:3]: I don't have much experience with json-schema, but had gotten a ping when I previously switched this to https and there's a PR pending to fix that back to http in the other manifests: https://github.com/opencontainers/image-spec/pull/931

**Review by sudo-bmitch — COMMENTED** [review]
Thanks for updating these @sajayantony!

File: schema/artifact-manifest-schema.json (hunk 1 above)
Comment (sajayantony) [reply schema/artifact-manifest-schema.json:3]: Thanks for pointing that out. Can keep this comment open and revert it once the other one is reverted.

**Comment by jdolitsky** [comment]
Thinking we maybe do not want to merge this into pr branch right now.. same rationale for dropping conformance test from distribtuion spec changeset. want to give people less green lines to look at. LGTM though otherwise 👍

**Comment by sudo-bmitch** [comment]
LGTM. Holding off on the approval for other PRs to get merged first.

**Comment by sudo-bmitch** [comment]
Closing since this is being handled upstream. Thanks for working on these!

**Pull request closed without merging**